mercator.php

```diff
@@ -324,3 +326,29 @@ function mangle_url( $url, $path, $orig_scheme, $site_id = 0 ) {
 
 	return $mangled;
 }
+
+/**
+ * Mangle the content URL to give our mapped domain
+ *
+ * @param string $url The complete home URL including scheme and path.
+ * @param string $path Path relative to the home URL. Blank string if no path is specified.
+ * @return string Mangled Content URL
+ */
+function mangle_content_url( $url, $path) {
+	//get site ID for use with getting mapped domain name
```

Comment:
Also, I'd drop this comment and the next (`pull in site aliases`) as they're not really adding much on top of the code itself here.